docs/src/tut/wfs/index.md

```diff
@@ -0,0 +1,1343 @@
+# {#wfs} Wisdom For Sale
+
+This module shows you how to build Wisdom For Sale, a command-line and web-based application that enables two participants (a seller and a buyer) to trade wisdom for tokens via a smart contract running on a consensus network, using Reach. Reach programs are built using a Docker container on your computer. If you need help installing Reach and its prerequisites then get started at our [Quick Installation Guide](#quickstart).
+
+Wisdom For Sale creates and funds two accounts (one for each participant), enables the seller and buyer to make a transaction, and then exits.
+
+# {#wfs-1} Learning Objectives
+
+The following learning objectives describe what you will be able to do as a result of completing this module.
+
+1. Define essential Reach terms.
+1. Build a simple Reach command-line DApp.
+1. Convert the command-line DApp into a vanilla Webapp.
+
+# {#wfs-2} Examine the transaction
+
+The following diagram represents the wisdom-for-sale transaction:
+
+![Seller and buyer roles](/tut/wfs/seller-buyer.png)
+
+This particular transaction took place on an Algorand devnet. The Algorand cryptocurrency standard token unit is the `ALGO`.
+As indicated by the final balances in the diagram, the seller received 0.006 ALGO less than the agreed upon price, and the buyer paid 0.003 ALGO more.
+These expenses represent `gas`, the cost of doing business on a consensus network.
```

Comment:
The general term is "fees" and the colloquial term used in some networks (ETH) is "gas". I would use "fees" and then have a margin note about how they are commonly called "gas"